03-TimeConversionApp/TimeConverter/ContentView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  ContentView.swift
+//  TimeConverter App
+//
+//  Created by Phil on 09/02/2023.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+  
+  @State private var inputValue = 60.0
+  @FocusState private var inputFocus
+  @State private var outputValue = 0.0
+  @State private var inputUnit = "Seconds"
+  @State private var outputUnit = "Minutes"
+  
+  private let options = ["Seconds", "Minutes", "Hours", "Days"]
```

Comment:
I would look at using an `enum` for this. Much safer when switching over the options etc. 

You can either conform the enum to `String` this means the `.rawValue` will be a string representation of the case name. Or you could have a property on the enum to provide a title. e.g:

```
var title: String {
    switch self {
    case .seconds: return "Seconds"
    etc
    }
}
```

Reply:
**Phil**
Which option would you go for out of the two?

**John Sanderson**
Thinking about it, id maybe go for a mix of both

```swift
enum TimeUnit: String {
    case seconds
    case minutes
    etc

    var displayName: String {
        self.rawValue.capitalised
    }
}
```
That way the displayName property is a bit more descriptive about what it does and returns you a capitalised version of the string but still using the rawValue conformance means you dont have to explicitly type "Seconds" etc, which means if you were to add new cases you would just get the display name for "free"

**Phil:**
Never seen a var inside an enum before. So I use it like TimeUnit.seconds.displayname?

**John Sanderson:**
Yeah exactly. You probably dont even need the TimeUnit bit, so just .seconds.displayName.

Actually you wouldnt even need that, you can continue to use the ForEach and I think you can just do
ForEach(options, id: \.self) {
    Text($0.displayName)
}